src/main/resources/data-repository-openapi.yaml

```diff
@@ -263,11 +273,13 @@ paths:
       - $ref: '#/parameters/Id'
       responses:
         200:
-          description: Job status
-          schema:
-            $ref: '#/definitions/JobModel'
```

Comment:
I think we should return the JobModel with this response. It lets us update the state of one job, instead of re-querying the jobs enumeration.